### PR TITLE
Correct findScrollableParent return type

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

The return type of `findScrollableParent` is wrong and has led to bugs in SharePoint Online.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/utilities/etc/utilities.api.md [hunk 1]
```diff
@@ -1,1234 +1,1234 @@
...
-export function warnDeprecations<P>(componentName: string, props: P, deprecationMap: ISettingsMap<P>): void;
-
-// @public
-export function warnMutuallyExclusive<P>(componentName: string, props: P, exclusiveMap: ISettingsMap<P>): void;
-
-
-// (No @packageDocumentation comment for this package)
-
-```
+## API Report File for "@uifabric/utilities"
```
Comment (dzearing) [inline packages/utilities/etc/utilities.api.md:1]: not sure why this entire file is different,... whitespace? line endings?

Comment (KevinTCoughlin) [reply packages/utilities/etc/utilities.api.md:1]: Probably. It took over 5 mins for all 31 commit hooks to finish a merge conflict resolution.

File: change/@fluentui-react-focus-2020-07-08-11-37-35-keco-scroll-return-type.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Additional HTMLElement casts have been added in a few places for better typing. This should not impact the public API surface.",
```
Comment (dzearing) [inline change/@fluentui-react-focus-2020-07-08-11-37-35-keco-scroll-return-type.json:3]: Tweaked your comments for you to be more readable in release notes.

File: packages/utilities/etc/utilities.api.md (hunk 1 above)
Comment (dzearing) [reply packages/utilities/etc/utilities.api.md:1]: weird! not sure.

**Pull request merged**